lib/bolt/cli.rb

```diff
@@ -96,6 +96,48 @@ def parse
       finalize_setup
     end
 
+    # Prints a welcome message when users first install Bolt and run `bolt`, `bolt help` or `bolt --help`
+    def welcome_message
+      bolt = <<~BOLT
+                   `.::-`
+              `.-:///////-.`
+           `-:////:.  `-:///:-  /ooo.                        .ooo/
+       `.-:///::///:-`   `-//:  ymmm-                        :mmmy  .---.
+      :///:-.   `.:////.  -//:  ymmm-                        :mmmy  +mmm+
+      ://.          ///.  -//:  ymmm--/++/-       `-/++/:`   :mmmy-:smmms::-
+      ://.          ://. .://:  ymmmdmmmmmmdo`  .smmmmmmmmh: :mmmysmmmmmmmms
+      ://.          ://:///:-.  ymmmh/--/hmmmy -mmmd/-.:hmmm+:mmmy.-smmms--.
+      ://:.`      .-////:-`     ymmm-     ymmm:hmmm-    `dmmm/mmmy  +mmm+
+      `-:///:-..:///:-.`        ymmm-     ommm/dmmm`     hmmm+mmmy  +mmm+
+         `.-:////:-`            ymmm+    /mmmm.ommms`   /mmmh:mmmy  +mmmo
+             `-.`               ymmmmmhhmmmmd:  ommmmhydmmmy`:mmmy  -mmmmdhd
+                                oyyy+shddhs/`    .+shddhy+-  -yyyo   .ohddhs
+
+
+      BOLT
+      example_cmd = if Bolt::Util.windows?
+                      "Invoke-BoltCommand -Command 'hostname' -Targets localhost"
+                    else
+                      "bolt command run 'hostname' --target localhost"
+                    end
+      prev_cmd = String.new("bolt")
+      prev_cmd << " #{@argv[0]}" unless @argv.empty?
+
+      message = <<~MSG
+      🎉 Welcome to Bolt #{VERSION}
+      😌 We're here to help bring order to the chaos
+      📖 Find our documentation at https://bolt.guide
+      🙋 Ask a question in #bolt on https://slack.puppet.com/
+      🤖 Contribute at https://github.com/puppetlabs/bolt/
```

Comment:
This emoji doesn't work for me in Terminal. It does show up in iTerm 2 though. Strange...

Reply:
Oo actually what does it look like when it doesn't work? If it just doesn't show up that seems fine, but if there's weird characters I can remove them. Or if there's some way to check if they're supported first we can do that

Reply:
Actually I zoomed in to take a screenshot and it _is_ a robot! Just not one that matches the rest of the emoji on my system. It looks like my font (Source Code Pro) just includes its own robot character that doesn't look much like a robot at 10pt.

![image](https://user-images.githubusercontent.com/115494/111212957-800ab380-858d-11eb-9639-56a01af203c7.png)

Reply:
Oh hmm...to be honest I don't love the robot for the "Contribute" line anyway, if we want to replace that...maybe with 🔩 ?